From 0419377c799899ac8e7c6b03d10c2a09f8f8431f Mon Sep 17 00:00:00 2001
From: David Shea <reallylongword@gmail.com>
Date: Tue, 3 Jan 2023 19:26:38 -0500
Subject: [PATCH 7/7] Separate C and python sources

This prevents setuptools from trying to install *.c and *.h files as
part of the python module.
---
 {src => nss}/__init__.py | 0
 setup.py                 | 2 +-
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename {src => nss}/__init__.py (100%)

diff --git a/src/__init__.py b/nss/__init__.py
similarity index 100%
rename from src/__init__.py
rename to nss/__init__.py
diff --git a/setup.py b/setup.py
index 137184b..9f2cada 100644
--- a/setup.py
+++ b/setup.py
@@ -42,7 +42,7 @@ nss_ssl_extension = Extension(
 
 setup(
     name="python-nss",
-    package_dir={"nss": "src"},
+    package_dir={"nss": "nss"},
     packages=["nss"],
     ext_modules=[nss_error_extension, nss_io_extension, nss_nss_extension, nss_ssl_extension],
 )
-- 
2.37.3

